### DISCUSSION TITLE
Litmus integration with Jenkins

### DISCUSSION THREAD

**Discussion by sravanthnag** [litmuschaos/litmus#3583] [Q&A] [answered]
Hi,
I have implemented the litmus chaos for a PoC purpose. To complete the intended PoC, I was looking for some references or help material if I can integrate the litmus chaos workflows to jenkins. I have found some material on web and on your portals but that was not fool proof and had some bugs in some steps. I would like to check with the dev community if there is any latest article or steps describing this integration.

Thanks in advance

**Comment by neelanjan00** [comment, marked as answer]
Hi @sravanthnag, thank you for asking this question! If possible can you repost this question in the Litmus community Slack channel in k8s Slack (https://k8s.slack.com)?

There's also this course on Udemy by @mikhailknyazev where he has discussed the integration of LitmusChaos with Jenkins, Terraform, Argo, Helm, and so on. It is a very good course if you want a detailed integration understanding of code snippets and concepts. I will leave the link here: https://www.udemy.com/course/configuring-kubernetes-for-reliability-with-litmuschaos/

Otherwise, feel free to discuss this on Slack, we do have community members who have integrated LitmusChaos with Jenkins.
